### PR TITLE
deletes some unused variables

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
See title

## Why It's Good For The Game
Variables that aren't used but exist in the code are rather confusing, so let's yeet them.

## Testing
Compiled, ran, made sure I didn't break genetics, realize old coders were not very smart

### CODE REVIEW

File: code/game/mecha/equipment/tools/work_tools.dm [hunk 1]
```diff
@@ -330,8 +330,6 @@
 	name = "cable layer"
 	desc = "Equipment for engineering exosuits. Lays cable along the exosuit's path."
 	icon_state = "mecha_wire"
-	var/datum/event/event
```
Comment (GDNgit) [inline code/game/mecha/equipment/tools/work_tools.dm:333]: this one is especially concerning

Comment (farie82) [reply code/game/mecha/equipment/tools/work_tools.dm:333]: Oh god, I remember this one yeah. A mecha tool trying to make an event... forgot to remove it back when I fixed that shit 😅

**Review by Sirryan2002 — APPROVED** [review]
code looks good

File: code/game/mecha/equipment/tools/work_tools.dm (hunk 1 above)
Comment (Fox-McCloud) [reply code/game/mecha/equipment/tools/work_tools.dm:333]: But why though.

**Review by Fox-McCloud — APPROVED** [review]

**Pull request merged**